scss/variables.scss

```diff
@@ -7,6 +7,7 @@ $total-width: 1366px;
 $black-default: #000;
 $blue: #00ADEE;
 $green-lemon: #BED23C;
+$dark-green: #74830f;
```

Comment:
Remember to uppercase the hex codes in order to mantain a standard, sorry for the nitpick :sweat_smile: